### PR TITLE
macOS, CPU-only

### PR DESCRIPTION
**Pull request by JanX2**
Small patch required to run on my macOS, CPU-only setup.

### CODE REVIEW

File: test.lua [hunk 1]
```diff
@@ -7,8 +7,8 @@
 
 require 'torch'
 require 'nn'
-require 'cunn'
-require 'cudnn'
+--require 'cunn' -- Done conditionally, if we are using GPU.
```
Comment (syhw) [inline test.lua]: You should just delete these two lines, not comment out. 👍

File: data/utils/convert-arpa.lua [hunk 2]
```diff
@@ -64,7 +64,7 @@ assert(ofarpa, string.format('cannot open <%s> for writing', opt.arpadst))
 -- src arpa
 local ifarpa
 if opt.arpasrc:match('%.z$') or opt.arpasrc:match('%.gz$') then
-   ifarpa = io.popen(string.format('zcat %s', opt.arpasrc))
```
Comment (VitaliyLi) [inline data/utils/convert-arpa.lua]: Could you please move it in a separate commit?

Comment (JanX2) [reply data/utils/convert-arpa.lua]: Done.

**Comment by vineelpratap** [comment]
Added in https://github.com/facebookresearch/wav2letter/pull/42
